### PR TITLE
Add hacktoberfest blog post

### PR DESCRIPTION
**Pull request by mauromorales**

### CODE REVIEW

File: content/en/blog/hacktoberfest-2023.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  - Initramfs
+  - Dracut
+  - Immutability
+- Containers and Dockerfiles management
+- Kubernetes
+- Static Site Generators
+- Edge computing
+- User Interfaces
+  - Web
+  - Command Line
```
Comment (mauromorales) [inline content/en/blog/hacktoberfest-2023.md:51]: I'm not sure if this is the best way to present this, do you have a better idea? I just want to make it clear that there are multiple areas for learning/contributing in our project

Comment (jimmykarily) [reply content/en/blog/hacktoberfest-2023.md:51]: I think it's fine.

File: content/en/blog/hacktoberfest-2023.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Plus you can also learn about unique Kairos related projects like:
+
+- Luet package manager
+- Immucore
+- EdgeVPN
```
Comment (jimmykarily) [inline content/en/blog/hacktoberfest-2023.md:57]:
```suggestion
- [Luet package manager](https://github.com/mudler/luet)
- [Immucore](https://github.com/kairos-io/immucore/)
- [EdgeVPN](https://mudler.github.io/edgevpn/)
```

**Review by jimmykarily — APPROVED** [review]
Good job!

**Pull request merged**